example/dynamicbox_server.py

```diff
@@ -15,10 +15,9 @@
    $ pvget -m foo:list
 """
 
-from __future__ import print_function
```

Comment:
Is this deleted just because it is unnecessary as there's no print() calls in this file? It feels like something outside the strict scope of this PR.